tests/actions.go

```diff
@@ -315,6 +341,12 @@ func (oa *operatorActions) CleanTidbCluster(info *TidbClusterInfo) error {
 		}
 	}
 
+	// delete all jobs
+	allJobsSet := label.Label{}.Instance(info.ClusterName).String()
```

Comment:
```suggestion
	allJobsSet := label.New().Instance(info.ClusterName).String()
```

Reply:
the namespace contains some jobs which label `app.kubernetes.io/managed-by: Tiller`.
these jobs cannot be select by `label.New()`